drop-in/src/main/java/com/adyen/checkout/dropin/ui/action/ActionComponentDialogFragment.kt

```diff
@@ -156,6 +167,11 @@ internal class ActionComponentDialogFragment : DropInBottomSheetDialogFragment()
                 Logger.d(TAG, "Flow was cancelled by user")
                 onBackPressed()
             }
+            is PermissionException -> {
+                val askedPermission = (componentError.exception as PermissionException).askedPermission
```

Comment:
This cast shouldn't be needed is you use `when (val exception = componentError.exception) {`